corehq/form_processor/submission_post.py

```diff
@@ -322,9 +330,35 @@ def run(self):
                 elif instance.is_error:
                     submission_type = 'error'
 
+            self._log_form_completion(instance, submission_type)
+
             response = self._get_open_rosa_response(instance, **openrosa_kwargs)
             return FormProcessingResult(response, instance, cases, ledgers, submission_type)
 
+    def _log_form_details(self, form):
+        attachments = form.attachments if hasattr(form, 'attachments') else {}
+
+        logger.info('Received Form %s with %d attachments',
+            form.form_id, len(attachments))
+
+        for index, (name, attachment) in enumerate(attachments.items()):
+            attachment_msg = 'Form %s, Attachment %s: %s'
+            attachment_props = [form.form_id, index, name]
+
+            if hasattr(attachment, 'has_size') and attachment.has_size():
+                attachment_msg = attachment_msg + ' (%d bytes)'
+                attachment_props.append(attachment.raw_content.size)
+
+            logger.info(attachment_msg, *attachment_props)
+
+    def _log_form_completion(self, form, submission_type):
+        # Orig_id only exists on SQL forms
```

Comment:
This comment is a bit misleading:
```suggestion
        # `orig_id` doesn't exist on all couch forms, only XFormError and XFormDeprecated
```

Reply:
Thanks Simon! Fixed.